packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -38,9 +38,26 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
     ? () => alert('you are using a unsupported browser')
     : undefined;
 
+  const onPermissionsTroubleshootingClick = queryArgs.useCallReadiness
+    ? (permissionsState: { camera: PermissionState; microphone: PermissionState }) => {
```

Comment:
Do you really need to strongly type this if all you do is `alert`? This can just be `permissionState: unknown`.